classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :age, :name, :id, :parent_permission
+
+  def initialize(age:, name: 'Unknow', parent_permission: true)
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] The function you defined is `of_age?`, not `is_of_age?`. Kindly correct that.

Reply:
Done.